doc/manpages/man1/nbp.1.md

```diff
@@ -20,7 +20,10 @@ be advertised.
 registered on the AppleTalk internet. *nbpname* is parsed by
 **nbp_name**(3). An \`*=*' for the *object* or *type* matches anything,
 and an \`*\**' for *zone* means the local zone. The default values are
-taken from the NBPLKUP environment variable, parsed as an *nbpname*.
+taken from the NBPLKUP environment variable, parsed as an *nbpname*.  If
+-s is specified, output is printed in a script-friendly format: for each
+response, first the address is printed, followed by a single space, followed 
+by the name and type, followed by a linefeed.
```

Comment:
Suggestion: It would be great to have this in a separate paragraph. I think it reads better that way, plus it makes localization with gettext (to Japanese) easier.